### PR TITLE
New package: QXYZLabs.Sanctum version 1.8.2

### PR DESCRIPTION
**Pull request by SpecterShell**
Created by [🥟 Dumplings](https://github.com/SpecterShell/Dumplings).
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/184971)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335226
> 2024-10-22 07:04:01.651 [CLI ] ShellExecute installer failed: 2

Automated error analysis suggests -1978335226 may mean Running ShellExecute failed

(Automated response - build 897.)

**Review by ItzLevvie — APPROVED** [review]

**Comment by SpecterShell** [comment]
Installs normally in my VM. @stephengillie

image

File: manifests/q/QXYZLabs/Sanctum/1.8.2/QXYZLabs.Sanctum.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+PackageIdentifier: QXYZLabs.Sanctum
+PackageVersion: 1.8.2
+InstallerType: nullsoft
+Scope: user
+UpgradeBehavior: install
+Protocols:
+- sanctum
+ReleaseDate: 2024-10-21
+Installers:
```
Comment (stephengillie) [inline manifests/q/QXYZLabs/Sanctum/1.8.2/QXYZLabs.Sanctum.installer.yaml:12]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
I didn't get a screenshot of the error, apologies.

**Comment by stephengillie** [comment]
With the dependency, verify:
image
image

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by SpecterShell** [comment]
https://github.com/microsoft/winget-pkgs/pull/184971#issuecomment-2434141788

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by SpecterShell** [comment]
@stephengillie https://github.com/microsoft/winget-pkgs/pull/184971#issuecomment-2434141788

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
Hi @SpecterShell,

A number of errors popped up during manual validation. Should `Nvidia.CUDA` or a similar package be a dependency?

image
image

Or libcrypto?
image
image
image

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335226
> 2024-11-04 12:06:09.668 [CLI ] ShellExecute installer failed: 2
> MSI (s) (04:9C) [12:04:48:801]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'.
> MSI (s) (04:9C) [12:04:48:801]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'.
> MSI (s) (04:9C) [12:04:48:801]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'.
> MSI (s) (04:9C) [12:04:48:801]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'.
> MSI (s) (04:9C) [12:04:48:801]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.
> Property(S): OutOfDiskSpace = 0
> Property(S): OutOfNoRbDiskSpace = 0
> Property(S): PrimaryVolumeSpaceAvailable = 0
> Property(S): PrimaryVolumeSpaceRequired = 0
> Property(S): PrimaryVolumeSpaceRemaining = 0
> MSI (s) (04:84) [12:04:50:411]: PROPERTY CHANGE: Adding OutOfDiskSpace property. Its value is '0'.
> MSI (s) (04:84) [12:04:50:411]: PROPERTY CHANGE: Adding OutOfNoRbDiskSpace property. Its value is '0'.
> MSI (s) (04:84) [12:04:50:411]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceAvailable property. Its value is '0'.
> MSI (s) (04:84) [12:04:50:411]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRequired property. Its value is '0'.
> MSI (s) (04:84) [12:04:50:411]: PROPERTY CHANGE: Adding PrimaryVolumeSpaceRemaining property. Its value is '0'.

Automated error analysis suggests -1978335226 may mean Running ShellExecute failed

(Automated response - build 897.)

**Comment by stephengillie** [comment]
Manual Validation ended with a normal install. 
image
(Not sure why it starts half off of the VM screen, but I"m assuming that's normal program operation.)

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**